src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Range.cs

```diff
@@ -123,12 +129,14 @@ private BoundExpression LiftRangeExpression(BoundRangeExpression node, BoundExpr
                 value: conditionalExpression,
                 type: node.Type);
 
-            BoundExpression getIndexFromPossibleNullable(BoundExpression arg)
+            // [return: NotNullIfNotNull("arg")]
```

Comment:
intentional?

Reply:
Yes, unfortunately one cannot put attributes on local functions.

---
In reply to: [387364650](https://github.com/dotnet/roslyn/pull/42068#discussion_r387364650) [](ancestors = 387364650)

Reply:
it is now possible to put attributes on local functions

Reply:
> it is now possible to put attributes on local functions

You mean in C# 8?

Reply:
Ah, good point. For some reason, I tought we're using LangVersion preview because of something Jared said. I'll confirm with him.

---
In reply to: [391947417](https://github.com/dotnet/roslyn/pull/42068#discussion_r391947417) [](ancestors = 391947417)